samples/tests/Console.js

```diff
@@ -30,3 +50,30 @@ console.log(longstr);
 console.log("\nTesting a string too long...");
 longstr += '6';  // length 256
 console.log(longstr);
+
+console.log("\nTesting console.assert... ");
+console.assert(true);
+console.log("assert(true): expected result nothing");
+
+try {
+    console.assert(false, "My lover!");
+} catch (e) {
+    console.log("assert(false, 'My lover!'): expected result throw error");
+    console.log("ERROR: " + e.name + " - " + e.message);
+}
+
+console.assert(consoleStr);
+console.log("assert(string): expected result [ERROR] invalid parameters");
+
+console.assert(false, longstr);
+console.log("assert(false, longstr): " +
+            "expected result [ERROR] string is too long");
+
+console.assert(false, consoleNum);
+console.log("assert(false, Num): " +
+            "expected result [ERROR] message parameter must be a string");
+
+console.log("\nTesting constructor to create...");
+var stdout, stderr;
+var myConsole = new Console(stdout, stderr);
+myConsole.log("create new console as 'myConsole'");
```

Comment:
This doesn't work and I don't think we've documented the constructor yet so I don't think I would write tests for it.

In any case, we have no way for you to provide valid streams to pass the constructor at this point, and it's not going to work to give it undefined variables like this.